CreateTable assignment mismatch

go-getting this repo produces a compiler error:

```
../../robinjoseph08/go-pg-migrations/setup.go:57:9: assignment mismatch: 2 variables but 1 values
```

go-pg/pg#957 changed the signature of `CreateTable` to omit the `Result` and only return an error, so `createTable` in `setup.go` should be changed to just return `orm.CreateTable(...)`.

--- a/setup.go
+++ b/setup.go
@@ -1,8 +1,11 @@
 package migrations
 
-import "github.com/go-pg/pg/orm"
+import (
+	"github.com/go-pg/pg"
+	"github.com/go-pg/pg/orm"
+)
 
-func ensureMigrationTables(db orm.DB) error {
+func ensureMigrationTables(db *pg.DB) error {
 	exists, err := checkIfTableExists("migrations", db)
 	if err != nil {
 		return err
@@ -52,8 +55,7 @@ func checkIfTableExists(name string, db orm.DB) (bool, error) {
 	return count > 0, nil
 }
 
-func createTable(model interface{}, db orm.DB) error {
+func createTable(model interface{}, db *pg.DB) error {
 	opts := orm.CreateTableOptions{IfNotExists: true}
-	_, err := orm.CreateTable(db, model, &opts)
-	return err
+	return db.CreateTable(model, &opts)
 }

